src/core/map.js

```diff
@@ -757,6 +758,12 @@ geo.map = function (arg) {
   this.interactor(arg.interactor || geo.mapInteractor());
   this.clock(arg.clock || geo.clock());
 
+  if (arg.autoResize) {
+    $(window).resize(function () {
+      m_this.resize(0, 0, m_node.width(), m_node.height());
```

Comment:
are we always going to start from the 0, 0 offset? I guess that's true for the most use-cases.